docs/developers/v2/yswaps.md

```diff
@@ -0,0 +1,194 @@
+# ySwaps
+
+ySwaps is used to **abstract** the token trading logic from the strategies harvest method, which means strategies can focus on farming tokens and don't have to handle tokens trading logic and paths. The "swappers" smart contracts allows trading optimizations and new paths to be added since the code is not hardcoded in strategies.
+
+- [ySwaps contracts repository](https://github.com/yearn/yswaps)
+- [ySwaps scripts repository](https://github.com/yearn/yswaps-app)
+
+![yswaps-intro-diagram](https://i.imgur.com/VynhCU4.png)
+
+## Glossary
+- `sms`: Strategists multisig
+- `tokenIn`:  Token we have and we want to trade/swap
+- `toeknOut`: Token we want to obtain on the swap
+- `hopToken`: Token we use as a middle step between tokenIn and tokenOut
+- `tf`: Trade Factory
+
+## Components
+
+- [Swappers](#Swappers)
+- [Trade Factory](#Trade-Factory)
+- [Solvers](#Solvers)
+- [Dexes Libraries](#Dexes-Libraries)
+- [Execute script](#Execute-script)
+- [Extra toolings](#Extra-toolings)
+- [Procedures](#Procedures)
+- [Keep3r Jobs](#Keep3r-Jobs)
```

Comment:
these links do not work. Try without caps, ie:

`#trade-factory` instead of `#Trade-Factory`

Reply:
fixed, they are working now!